tests/feed.js

```diff
@@ -95,7 +95,7 @@ test('replicate to hypercore from archiver', function (t) {
 })
 
 test('replicate two hypercores from archiver', function (t) {
-  t.plan(6)
+  t.plan(4)
```

Comment:
Ah ya, 6 is correct because `thisArchives.on('archived')` should emit twice, once for each feed. 

This is why I think the double append is still buggy. The second feed never emits archived. Definitely confused about this bit so please double check my logic =).